docs/concepts.md

```diff
@@ -4,7 +4,7 @@ This document explains some core terms used throughout the documentation.
 
 ## Controller and agents
 
-While it is possible to run benchmarks directly from CLI, in its nature Hyperfoil is a distributed tool with master-slave architecture. **Controller** has the master role; this is a [Vert.x]((https://vertx.io))-based server with REST API. When a benchmark is started controller deploys agents (according to the benchmark definition), pushes the benchmark definition to these agents and orchestrates benchmark phases. **Agents** execute the benchmark, periodically sending statistics to the controller. This way the controller can combine and evaluate statistics from all agents on the fly. When the benchmark is completed all agents terminate.
+While it is possible to run benchmarks directly from CLI, in its nature Hyperfoil is a distributed tool with leader-follower architecture. **Controller** has the master role; this is a [Vert.x]((https://vertx.io))-based server with REST API. When a benchmark is started controller deploys agents (according to the benchmark definition), pushes the benchmark definition to these agents and orchestrates benchmark phases. **Agents** execute the benchmark, periodically sending statistics to the controller. This way the controller can combine and evaluate statistics from all agents on the fly. When the benchmark is completed all agents terminate.
```

Comment:
Still contains 'master role' - not sure if such expression is on the whitelist.

Reply:
I suggest`master role` can be swapped with `leader role` as well.
I think [whitelist](https://tools.ietf.org/id/draft-knodel-terminology-00.html#rfc.section.1.2) also comes up as an antiquated term. Best avoid that as well and use a suggested alternative.